src/templates/admin/core/article.html

```diff
@@ -22,12 +22,13 @@ <h2>{{ article.title }}</h2>
         </div>
     </div>
 
+    {% if not journal_settings.general.hide_review_data_pre_release or article.peer_reviews_for_author_consumption %}
```

Comment:
With the latest change, did you decide to ignore whether an article has passed the review stage? It will no longer show the block once the article has been accepted or declined, as it did before. Is that what you want?